install/0000_90_machine-api-operator_03_servicemonitor.yaml

```
@@ -22,4 +22,43 @@ spec:
   selector:
     matchLabels:
       k8s-app: machine-api-operator
-
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  namespace: openshift-machine-api
+  name: machine-api-controllers
+  labels:
+    k8s-app: controller
+spec:
+  namespaceSelector:
+    matchNames:
+      - openshift-machine-api
+  selector:
+    matchLabels:
+      k8s-app: controller
+  endpoints:
+  - port: machine-metrics
+    bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 30s
+    tlsConfig:
+      caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
+      serverName: machine-api-controllers.openshift-machine-api.svc
+  - port: controller-metrics
```

If we are changing the port name
```suggestion
  - port: machineset-metrics
```